tasks/users.ts

```diff
@@ -9,148 +9,118 @@ import {
-      if (hre.network.name === "rsk") {
-        const oracleEthUsd = (await hre.ethers.getContract(PriceFeedContractNames.ETH_USD, deployer)) as SovrynSwapPriceFeed;
-        if (!(await isSameOwner(owner, oracleEthUsd))) {
-          console.log(`transfer ownership on contract ${PriceFeedContractNames.ETH_USD} to ${owner}`);
-          await (await oracleEthUsd.transferOwnership(owner)).wait();
-        }
+    const contracts = await getContractsToTransferOwnership(hre);
 
-        const oracleBtcEth = (await hre.ethers.getContract(PriceFeedContractNames.BTC_ETH, deployer)) as SovrynSwapPriceFeed;
-        if (!(await isSameOwner(owner, oracleBtcEth))) {
-          console.log(`transfer ownership on contract ${PriceFeedContractNames.BTC_ETH} to ${owner}`);
-          await (await oracleBtcEth.transferOwnership(owner)).wait();
-        }
+    for (const contract of contracts) {
+      await (await contract.connect(deployer).transferOwnership(timelock.address)).wait();
+    }
 
-        const oracleEurUsd = (await hre.ethers.getContract(PriceFeedContractNames.EUR_USD, deployer)) as FiatToFiatPseudoOracleFeed;
-        if (!(await isSameOwner(owner, oracleEurUsd))) {
-          console.log(`transfer ownership on contract ${PriceFeedContractNames.EUR_USD} to ${owner}`);
-          await (await oracleEurUsd.transferOwnership(owner)).wait();
-        }
+    console.log("owner changed to timelock");
+  });
 
-        const oracleGbpUsd = (await hre.ethers.getContract(PriceFeedContractNames.GBP_USD, deployer)) as FiatToFiatPseudoOracleFeed;
-        if (!(await isSameOwner(owner, oracleGbpUsd))) {
-          console.log(`transfer ownership on contract ${PriceFeedContractNames.GBP_USD} to ${owner}`);
-          await (await oracleGbpUsd.transferOwnership(owner)).wait();
-        }
+  task("users:queue-change-owner-to-deployer").setAction(async (args_, hre) => {
+    const contracts = await getContractsToTransferOwnership(hre);
 
-        const oracleXauUsd = (await hre.ethers.getContract(PriceFeedContractNames.XAU_USD, deployer)) as FiatToFiatPseudoOracleFeed;
-        if (!(await isSameOwner(owner, oracleXauUsd))) {
-          console.log(`transfer ownership on contract ${PriceFeedContractNames.XAU_USD} to ${owner}`);
-          await (await oracleXauUsd.transferOwnership(owner)).wait();
-        }
-      }
+    const deployer = await getDeployer(hre);
+    const proposer = await getProposer(hre);
 
-      const pools = await getPools(hre);
-      for (const pool of pools) {
-        const uniOracle = await getUniswapPairOracle(hre, pool[0].name, pool[1].name);
-        if (!(await isSameOwner(owner, uniOracle))) {
-          console.log(`transfer ownership on uniswap pair oracle ${pool[0].name}-${pool[1].name} to ${owner}`);
-          await (await uniOracle.transferOwnership(owner)).wait();
-        }
-      }
+    const recorder = await defaultTimelockRecorder(hre, { executionStartInDays: null, singer: proposer });
```

Comment:
```suggestion
    const recorder = await defaultTimelockRecorder(hre, { executionStartInDays: null, signer: proposer });
```

I see this typo in more other places so you better find & replace all of them